add instruction for using parameters and sorting for stored script (#1834)
opensearch-project/documentation-website: _api-reference/script-apis/exec-stored-script.md

| _index | String | Index that contains the document. |
| _id | String | Document ID. |
| _score | Float | Document's relevance score. |
| fields | Object | Fields and their value returned from the script. |

## Execute Painless stored script with parameters
If you want to pass different parameters to the script each time of running query, define `params` in script_fields

### Sample request

The following request runs the stored script that was created in [Create or update stored script]({{site.url}}{{site.baseurl}}/api-reference/script-apis/create-stored-script/). The script sums the ratings for each book, multiply summed-up value by `multiplier` param and displays the result in the output.

* The script's target is the `books` index.

* The `"match_all": {}` property value is an empty object indicating to process each document in the index.

> * The `total_ratings` field value is the result of the `multiplier-script` execution. See  [Create or update stored script]({{site.url}}{{site.baseurl}}/api-reference/script-apis/create-stored-script/).

Reviewer: ```suggestion
* The `total_ratings` field value is the result of the `multiplier-script` execution. See [Create or update stored script]({{site.url}}{{site.baseurl}}/api-reference/script-apis/create-stored-script/).
```

* `"multiplier": 2` in `params` field is a variable passed into the stored script `multiplier-script`.

````json
GET books/_search
{
   "query": {
    "match_all": {}
  },
  "script_fields": {
    "total_ratings": {
      "script": {
        "id": "multiplier-script", 
        "params": {
          "multiplier": 2
        }        
      }
    }
  }
}
````

> ### Sample response

Reviewer: ```suggestion
### Example response
```

| _index | String | Index that contains the document. |
| _id | String | Document ID. |
| _score | Float | Document's relevance score. |
| fields | Object | Fields and their value returned from the script. |

> ## Execute Painless stored script with parameters

Reviewer 2: "Running a Painless stored script with parameters"?

> ## Execute Painless stored script with parameters

Reviewer: ```suggestion
## Running a Painless stored script with parameters
```

To pass different parameters to the script each time when running a query, define `params` in `script_fields`.

### Example

> The following request runs the stored script that was created in [Create or update stored script]({{site.url}}{{site.baseurl}}/api-reference/script-apis/create-stored-script/). The script sums the ratings for each book, multiplies the summed-up value by the `multiplier` param, and displays the result in the output.

Reviewer 2: ```suggestion
The following request runs the stored script that was created in [Create or update stored script]({{site.url}}{{site.baseurl}}/api-reference/script-apis/create-stored-script/). The script sums the ratings for each book, multiplies the summed value by the `multiplier` parameter, and displays the result in the output.
```

* The script's target is the `books` index.

> * The `"match_all": {}` property value is an empty object indicating processing each document in the index.

Reviewer 3: Is this the meaning? "... indicating that it processes each document in the index."

> * The `"match_all": {}` property value is an empty object indicating processing each document in the index.

Reviewer: ```suggestion
* The `"match_all": {}` property value is an empty object indicating that it processes each document in the index.
```

> * The `"match_all": {}` property value is an empty object indicating that it processes each document in the index.

Reviewer 2: ```suggestion
* The `"match_all": {}` property value is an empty object, indicating that it processes each document in the index.
```

> * The `total_ratings` field value is the result of the `multiplier-script` execution. See [Create or update stored script]({{site.url}}{{site.baseurl}}/api-reference/script-apis/create-stored-script/).

Reviewer 2: Please update link text based on previous heading change.

> * The `total_ratings` field value is the result of the `multiplier-script` execution. See [Create or update stored script]({{site.url}}{{site.baseurl}}/api-reference/script-apis/create-stored-script/).

Reviewer: ```suggestion
* The `total_ratings` field value is the result of the `multiplier-script` execution. See [Creating or updating a stored script with parameters]({{site.url}}{{site.baseurl}}/api-reference/script-apis/create-stored-script/).
```
